packages/core/src/service/CacheService.ts

```diff
@@ -114,7 +114,9 @@ export class CacheService {
 	 */
 	private async getCacheFileUri(): Promise<string> {
 		return (this.#cacheFilePath ??= new Uri(
-			`symbols/${await this.project.externals.crypto.getSha1(this.project.projectRoot)}.json.gz`,
+			`symbols/${await this.project.externals.crypto.getSha1(
+				this.project.projectRoots[0],
```

Comment:
I think using a combination of all roots here would be appropriate (something like `projectRoots.sort().join(':')`? ). Imo the expected behavior for multi-root is that we can reference resources across roots, so the same symbol table should be shared across (and therefore cached for) all roots. Though we'll have to think about how to support multi version properly.